terraform/environments/delius-iaps/templates/iaps-EC2LaunchV2.yaml.tftpl

```diff
@@ -87,6 +87,15 @@ tasks:
         $token = invoke-restmethod -Headers @{"X-aws-ec2-metadata-token-ttl-seconds"=3600} -Method PUT -Uri http://169.254.169.254/latest/api/token
         $instanceId = invoke-restmethod -Headers @{"X-aws-ec2-metadata-token" = $token} -Method GET -uri http://169.254.169.254/latest/meta-data/instance-id
         Add-Computer -DomainName "${delius_iaps_ad_domain_name}" -Credential $domainJoinCredential -NewName $instanceId -Force
+
+        # Install AD Management Tools
+        Install-WindowsFeature -Name RSAT-AD-PowerShell
+        Install-WindowsFeature -Name RSAT-ADDS-Tools
+
+        # Allow Domain Users to connect via RDP and give them local admin rights
+        Add-LocalGroupMember -Group "Remote Desktop Users" -Member "Domain Users@${delius_iaps_ad_domain_name}"
```

Comment:
Do administrators also need to be members of remote desktop users?

Reply:
corrected